### PR TITLE
Rename root folders

### PR DESCRIPTION
**Pull request by jamesvclements**
We don't have an issue for this, but we decided to rename the root folders to `UI` and `API ` respectively. This adds more clarity, since realistically the `ui` runs off its own server, and the `api` i serving up the BarTop API elsewhere.

It also includes README adjustments to clarify the README structure and dev instructions.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ cache:
   - node_modules
   - server/node_modules
   - client/node_modules
-after_success: concurrently 'cd server/ && npm run coverage' 'cd client/ && npm run coverage'
+after_success: concurrently 'cd bartop-api/ && npm run coverage' 'cd bartop-ui/ && npm run coverage'
```
Comment (dpopp07) [inline .travis.yml]: Has the Travis build really been using concurrently this whole time? How is it getting that package if we uninstalled it? Should we take that out?

Comment (jamesvclements) [reply .travis.yml]: Weird - yeah it definitely needs to be taken out, not sure how it wasn't breaking 🤔

File: README.md [hunk 2]
```diff
@@ -3,42 +3,62 @@
...
+`npm start`
+The API will be available for requests at http://localhost:3001 (if you set BARTOP_API_PORT to 3001 in your `.env` file).
 
-### Running the App Locally
-BarTop runs on two servers - one serving the client and one serving our API.
-* `npm run server` - starts the API server at http://localhost:3001
-* `npm run client` - starts the client server at http://localhost:3000
+#### Test the API
+The API test suite consists of both unit tests and integration tests.
+To run the full test suitem run:
```
Comment (jamesvclements) [inline README.md]: suite*

Comment (dpopp07) [reply README.md]: Thanks, good catch

File: .travis.yml (hunk 1 above)
Comment (dpopp07) [reply .travis.yml]: Yeah that's what I was thinking

File: README.md [hunk 3]
```diff
@@ -3,42 +3,95 @@
...
+*These instructions assume you're in the `/bartop-ui` directory*
 
-### Starting the Database
-BarTop uses RethinkDB for its persistence layer. Running this app locally requires a local of installation of RethinkDB.
+#### Install the dependencies
+`npm install`
 
-[Install RethinkDB.](https://www.rethinkdb.com/docs/install/)
+#### Environment Variables
+The API is configured using a `.env` file. Use the `.env.example` file as a template to create a local version.
```
Comment (dpopp07) [inline README.md]: UI*

**Review by dpopp07 — APPROVED** [review]
Good work everyone

**Pull request merged**